lib/Twig/Error/Syntax.php

```diff
@@ -17,4 +17,38 @@
  */
 class Twig_Error_Syntax extends Twig_Error
 {
+    /**
+     * Tweaks the error message to include suggestions.
+     *
+     * @param string $name  The original name of the item that does not exist
+     * @param array  $items An array of possible items
+     */
+    public function addMessageSuggestions($name, array $items)
+    {
+        if (!$alternatives = self::computeAlternatives($name, $items)) {
+            return;
+        }
+
+        $this->rawMessage .= sprintf('. Did you mean "%s"?', implode('", "', $alternatives));
```

Comment:
should we really add the dot here ? It means that the message without alternatives does not have a final dot